bootstrap/Platform_Management/Job_Runner.groovy

```diff
@@ -22,6 +22,9 @@ build job: 'Platform_Management/Load_Cartridge_List'
 
 // Setup the Gerrit ACL
 build job: 'Platform_Management/Setup_Gerrit'
+
+// Setup the Pluggable Library
+build job: 'Platform_Management/Setup_Pluggable_Library'
```

Comment:
I don't think this should be brought in as part of this PR since it is covered under here anyway: https://github.com/Accenture/adop-platform-management/pull/30/commits

So once the above is merged in, this branch can be rebased upon master.